src/datepicker/datepicker.js

```diff
@@ -33,6 +35,13 @@ angular.module('ui.bootstrap.datepicker', ['ui.bootstrap.dateparser', 'ui.bootst
     self[key] = angular.isDefined($attrs[key]) ? (index < 6 ? $interpolate($attrs[key])($scope.$parent) : $scope.$parent.$eval($attrs[key])) : datepickerConfig[key];
   });
 
+  self.timelessJsonMode = angular.isDefined($attrs.timelessJsonMode) ?
+     $attrs.timelessJsonMode.toLowerCase() != 'false' && ($attrs.timelessJsonMode || true)
```

Comment:
Ah, I see - in any case, this can still be simplified to `self-timelessJsonMode = angular.isDefined($attrs.timelessJsonMode) ? $attrs.timelessJsonMode !== 'false' && $attrs.timelessJsonMode`

I don't think it should default to true, this should be opt-in.